web_poet/testing/exceptions.py

```diff
@@ -0,0 +1,14 @@
+class FieldMissing(AssertionError):
+    pass
+
+
+class FieldValueError(AssertionError):
+    pass
+
+
+class ItemValueError(AssertionError):
+    pass
+
+
+class FieldsUnexpected(AssertionError):
+    pass
```

Comment:
I wonder if we should rename `FieldMissing` and `FieldsUnexpected` to `MissingFieldError` and `UnexpectedFieldsError` for consistency.

Alternatively, rename all to something without an `Error` suffix but still consistent with each other (i.e. ending with a noun), e.g. `MissingField`, `UnexpectedFieldValue`, `UnexpectedItemValue`, `UnexpectedFields`.

Reply:
My logic was to use Field.. prefixes for field errors and Item prefix for item error, and omit "Error" because they'are all in .exceptions module.

Reply:
though I haven't omitted "Error", so probably the logic was different :)

Reply:
Anyways, happy to proceed with any naming.

Reply:
Would this be better? 

* FieldMissing
* FieldValueIncorrect
* ItemValueIncorrect
* FieldsUnexpected

Reply:
The reason to use Field vs Item prefixes was that there is a very clear separation between those exception types: Item.. can only be raised when `--web-poet-test-per-item` is passed, and Field.. can only be raised if it's not passed.

Reply:
Just to be clear, I don’t have a strong opinion against the proposed names. https://github.com/scrapinghub/web-poet/pull/129#discussion_r1106367476 would also work for me.